flake.nix

```diff
@@ -14,6 +14,18 @@
     dune-nix.inputs.flake-utils.follows = "flake-utils";
     flake-utils.url = "github:numtide/flake-utils";
   };
+  nixConfig = {
```

Comment:
Is `README-nix.md` up to date, accounting for this flow?  It looks like we're adding config here that is called out as suggestions in the readme.  Could you take a pass over that and prune any irrelevant content?